packages/react/src/components/InAppMessaging/hooks/useMessageProps/useMessageProps.ts

```diff
@@ -33,28 +32,16 @@ export default function useMessageProps(
     }
   }, [onDisplay, shouldRenderMessage]);
 
-  const hasPrimaryButton = !isEmpty(primaryButton);
-  const hasSecondaryButton = !isEmpty(secondaryButton);
-  const hasButtons = hasPrimaryButton || hasSecondaryButton;
-
-  const styles = useMemo(() => {
-    // prevent generating style if message rendering is delayed
-    if (!shouldRenderMessage) {
-      return null;
-    }
```

Comment:
There is no need to delay `styles` from being computed. This was copied over from the React Native version of `useMessageProps` where the delay was used to prevent calling the style utils until image layout dimensions were known.